src/osx.rs

```diff
@@ -0,0 +1,36 @@
+pub(crate) mod internal {
+    extern "C" {
+        pub fn IsMicrophoneOn(status: *mut std::ffi::c_int) -> std::ffi::c_int;
+        pub fn IsCameraOn(status: *mut std::ffi::c_int) -> std::ffi::c_int;
+    }
+
+    use crate::State;
+
+    pub struct MediaState;
+
+    impl MediaState {
+        pub fn new() -> Self {
+            Self
+        }
+
+        pub fn mic_status(&self) -> Option<State> {
+            let mut status = -1;
+
+            if unsafe { IsMicrophoneOn(&mut status as _) } != -1 {
```

Comment:
it should be 0. why did this change?

Reply:
I did not (at least intentionally), idk how it got changed. weird. Let me fix that